internal/k8s/crdcontroller.go

```diff
@@ -98,7 +101,7 @@ func (c *AviController) SetupAKOCRDEventHandlers(numWorkers uint32) {
 				namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(hostrule))
 				key := lib.HostRule + "/" + utils.ObjKey(hostrule)
 				if err := validateHostRuleObj(key, hostrule); err != nil {
-					utils.AviLog.Warnf("key: %s, msg: Error retrieved during validation of HostRule: %v", key, err)
+					utils.AviLog.Warnf("Error retrieved during validation of HostRule: %v", err)
```

Comment:
@swathinsankaran: Let's keep message format as `key: %s, msg: %s` to have better diagnosis. I missed this yesterday to comment.

Reply:
I have addressed the comment. Thanks